app/boards/shields/settings_reset/settings_reset.conf

```diff
@@ -1 +1,4 @@
-CONFIG_ZMK_BLE_CLEAR_BONDS_ON_START=y
+CONFIG_SETTINGS=y
+CONFIG_ZMK_SETTINGS_RESET_ON_START=y
+# Disable BLE so splits don't try to re-pair until normal firmware is flashed.
+CONFIG_ZMK_BLE=n
```

Comment:
I do wonder if this might lead to folks not being clear if the updated firmware flashed or not... Maybe a docs update to note that when you flash settings reset, the device won't be visible/connectable at all again until flashed with regular firmware?

Reply:
Yeah, that seems like it would be good to clarify. Will do.

Reply:
Done. How does that look?

Reply:
LGTM. Has this gotten thorough testing locally there?

Reply:
I've tested it with the NVS backend. The file backend is untested. I'm not sure what currently supported build (if any) could be used for that.